README.md

```diff
@@ -26,7 +26,7 @@ positional arguments:
 
 options:
   -h, --help            show this help message and exit
-  --format {averyL4731,avery5160,avery5161,avery5163,avery5167,avery5371,herma10003}, -f {averyL4731,avery5160,avery5161,avery5163,avery5167,avery5371,herma10003}
+  --format, -f {averyL4731,averyL4732,avery5160,avery5161,avery5163,avery5167,avery5371,herma4201,herma10003,herma4346}
```

Comment:
The `--format, -f` breaks with the format of the rest of the file. Maybe consider a dedicated PR with the format change.

And what have the averyL4732 and herma4201 to do with this PR?

Reply:
Yes, thats something weird with the history. Will adjust that.